solve for #148

gxstrom opened the pull request:
Solved Issue #648

gxstrom committed:
Add files via upload

NtemKenyor commented on modules/text_on_image/text_on_image_NUM_YOUR_NAME_.html:
```diff
@@ -0,0 +1,63 @@
+    <p class="white-text large-text">para3</p>
+</div>
+<!-- change the path for your use-->
+<img src="path1" alt="" class="top-left-image"> 
```
please add a real image from the image folder in this module section.

NtemKenyor commented on modules/text_on_image/text_on_image_NUM_YOUR_NAME_.html:
```diff
@@ -0,0 +1,63 @@
+<img src="path2" alt="" class="image-right-bottom">
+
+<style>
+    body {
```
you can avoid giving the entire body some styles because that may affect the entire website. You can instead: create a div and give it  a width of 100% so that it fits any parent tag. This is not mandatory though but just makes your module high-class.

NtemKenyor reviewed (commented):
Just make this few changes and I would gladly accept your PR. thank you.

NtemKenyor reviewed (changes requested):
Please attend to the review and make a submission. would be glad to accept.

gxstrom committed:
Update text_on_image_NUM_YOUR_NAME_.html

Thanks a lot for the code review, I made some minor changes from that. Hope it will be ok

NtemKenyor reviewed (approved):
hello
